compiler-rt/test/asan/TestCases/Windows/alloc_dealloc_mismatch.cpp

```diff
@@ -0,0 +1,26 @@
+// RUN: %clangxx_asan -O0 %s -o %t
+// RUN: %env_asan_opts=alloc_dealloc_mismatch=1 not %run %t 2>&1 | FileCheck %s --check-prefix=CHECK-MISMATCH
+// RUN: %env_asan_opts=alloc_dealloc_mismatch=0 %run %t 2>&1 | FileCheck %s --check-prefix=CHECK-SUCCESS
+
+// RUN: %clangxx_asan -O0 %s -o %t -DUSER_FUNCTION
+// RUN: not %run %t 2>&1 | FileCheck %s --check-prefix=CHECK-MISMATCH

```

Comment:
can you add the same with both DUSER_FUNCTION and  %env_asan_opts=alloc_dealloc_mismatch=0

Expected behavior is env override default options

Reply:
Ah, good catch. That wasn't happening _before_, but I've fixed up the PR [through this commit](https://github.com/llvm/llvm-project/pull/122990/commits/27bc1c7afb2c733180f9f2641685c92f9330c835) to ensure `ASAN_OPTS` continues to take precedence. Thank you.